autometa/config/databases.py

```diff
@@ -301,6 +301,7 @@ def configure(self):
 def main():
     import argparse
     import logging as logger
+    import multiprocessing as mp
```

Comment:
I wanted to make the argparse block standalone so that we can easily copy it to another file and run it.
If I remove import multiprocessing as mp and copy the arparse block and try to run it it gives me an error.